src/accounts-service-access.vala

```diff
@@ -143,11 +143,8 @@ public class AccountsServiceAccess : Object
             string user_path;
             if (user_path_variant.check_format_string ("(o)", true)) {
                 user_path_variant.get ("(o)", out user_path);
-#if HAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS
                 _user_proxy = yield new DBusProxy.for_bus (BusType.SYSTEM, DBusProxyFlags.GET_INVALIDATED_PROPERTIES, null, "org.freedesktop.Accounts", user_path, "com.lomiri.AccountsService.Sound");
-#else
-                _user_proxy = yield new DBusProxy.for_bus (BusType.SYSTEM, DBusProxyFlags.GET_INVALIDATED_PROPERTIES, null, "org.freedesktop.Accounts", user_path, "org.ayatana.AccountsService.Sound");
-#endif
+
```

Comment:
Only initialize _user_proxy if(def) HAS_LOMIRI_ACCTSERVICE_SOUND_SETTINGS. Otherwise, set to null.